utils/debug/remoteDebugger_test.go

```diff
@@ -15,7 +15,7 @@ import (
 func TestDebugger_RunTransaction(t *testing.T) {
 
 	// this code is mostly a sample code so we skip by default
-	t.Skip()
+	// t.Skip()
```

Comment:
@ramtinms if this is the case, would it make more sense to implement this as a [runnable example](https://go.dev/blog/examples)?

More info: https://pkg.go.dev/testing#hdr-Examples

Reply:
I think runnable examples are for a different purposes, this code is mostly for a boilerplate for when we have to debug mainnet and testnet, I think a better way might be move this to a README file inside the debug package.

Reply:
I'll create a follow-up PR for this, to remove it as a test, so you can leave this skipped without info for now.

Reply:
@ramtinms actually, with this PR we would like to remove *all* direct calls to `t.Skip()` and enforce that going forward, every skip should use `unittest.SkipUnless` instead.

Are you okay with me removing this test entirely for now, and then later you can add it back as a README? (It should be easy to recover the code from version control).

Reply:
yeah we could do that,